CHANGELOG.md

```diff
@@ -34,34 +41,7 @@
 
 ### Misc
 
-- Prevent ICE when parsing invalid attributes in `cfg_if!` macros [#5728](https://github.com/rust-lang/rustfmt/issues/5728)
```

Comment:
Moved up to FIXED section, as this was a bug producing an ice that was fixed vs. a miscellaneous change